src/index.ts

```diff
@@ -1,13 +1,27 @@
-import { getInput, info, setOutput } from "@actions/core";
+import { getBooleanInput, getInput, getMultilineInput, info, setOutput } from "@actions/core";
 import { context, getOctokit } from "@actions/github";
 import { Context } from "@actions/github/lib/context";
 import moment from "moment";
-import { fetchIssues, IssueData } from "./github/issuesParser";
+import { byNoComments, isNotFromAuthor, olderThanDays } from "./filters";
+import { fetchIssues } from "./github/issuesParser";
 
 const daysSinceDate = (date: string): number => {
     return moment().diff(moment(date), 'days')
 }
 
+const getFiltersFromInput = (): Filters => {
```

Comment:
are there any filters that may go directly to the query?

Reply:
Not for now, but we may add them eventually.

I just wanted to abstract the query object so we can manage the logic better.